config/sync/field.field.paragraph.stanford_media_caption.su_media_caption_caption.yml

```diff
@@ -18,7 +18,7 @@ field_name: su_media_caption_caption
 entity_type: paragraph
 bundle: stanford_media_caption
 label: Caption
-description: 'The caption text that appears beneath the media item.'
+description: 'A “caption” is a brief description of the media that appears below the item in a small, gray color font.'
```

Comment:
I'm going to make a change, because it's not guaranteed to be a "small, gray color font"